src/app/ipv/middleware.js

```diff
@@ -358,87 +341,54 @@ module.exports = {
       delete req.session.currentPageStatusCode;
     }
   },
+
+  // make considerations for handleEscapeAction pages with no req.body?.journey that defaults to `end`
   handleJourneyAction: async (req, res, next) => {
     const currentPageId = req.params.pageId;
+    const pagesUsingSessionId = [
+      "pyi-suggest-other-options",
+      "pyi-cri-escape",
+      "pyi-kbv-escape-m2b",
+      "pyi-escape-m2b",
+      "page-multiple-doc-check",
+    ];
+    const journeyActions = {
+      end: "end",
+      "address-current": "address-current",
+      "attempt-recovery": "attempt-recovery",
+      "build-client-oauth-response": "build-client-oauth-response",
+      next: "next",
+      "next/end": "end",
+      "next/f2f": "f2f",
+      "next/dcmaw": "dcmaw",
+      "next/bank-account": "bankAccount",
+      "next/passport": "ukPassport",
+      "next/driving-licence": "drivingLicence",
```

Comment:
I was hoping that we might be able to do away with all this mapping, given we are now validating journey events on the backend, and posting some invalid journey events isn't as risky as it was before

I expect we will still need a list of pages that need to default to back instead of next, but we should be able to either change the radio button forms so that the journey event is already in the right format, or update the journey map to match if we need to

Reply:
Good shout. I do agree, but was optimistic about changing it earlier.